### DISCUSSION TITLE
Check missing glyphs in the font file using Python API

### DISCUSSION THREAD

**Discussion by ml16685** [fontforge/fontforge#4814] [Q&A]
Hello,

Could you please advice the correct way to check, whether particular glyph
is presented in the font file or not, using FontForge Python API.

As I know, there is the needed functionality in the other FontForge scripting language:

InFont(arg) 

Is there something similar in the Python API version?

So far, the only solution I could find is try to index font[] and catch the TypeError exception:

    try:
        font[code]
    except TypeError:
        prrint('Missing glyph', code)

But it seems an awkward approach.

**Comment by haydenwong7bm** [comment]
`code in font`. See [`font.__contains__`](https://fontforge.org/docs/scripting/python/fontforge.html#fontforge.font.__contains__) in the documentation.
